CARL/source/Recognizer.cpp

```diff
@@ -241,100 +282,117 @@ namespace carl::action
                         {
                             for (size_t r = l + 1; r < m_templates.size(); ++r)
                             {
-                                averageDistances[idx] += calculateNormalizedSequenceDistance(m_templates[l], m_templates[r]);
+                                auto distance = calculateNormalizedSequenceDistance(m_templates[l], m_templates[r]);
+                                averageDistances[idx] += distance;
                             }
                         }
                         averageDistances[idx] /= templatesChooseTwo;
-                        m_tuning[idx] = 0.;
+                        m_tuning[idx] = NULL_TUNING;
                     }
                 }
 
-                constexpr double defaultTuningWeight{ 1. };
-                double t = m_templates.size() > 1 ? 1. / (templatesChooseTwo + 1) : defaultTuningWeight;
+                constexpr NumberT defaultTuningWeight{ 1. };
+                NumberT t = m_templates.size() > 1 ? 1. / (templatesChooseTwo + 1) : defaultTuningWeight;
                 for (size_t idx = 0; idx < m_tuning.size(); ++idx)
                 {
-                    m_tuning[idx] = t * DescriptorT::DEFAULT_TUNING[idx] +
-                        (1. - t) * (1. / std::max(averageDistances[idx], kAverageDistanceEpsilon));
+                    m_tuning[idx] = 1;/*t * DescriptorT::DEFAULT_TUNING[idx] +
+                        (1. - t) * (1. / std::max(averageDistances[idx], kAverageDistanceEpsilon));*/
```

Comment:
Auto-tuning is essentially disabled at the moment. This situation will not last, but it remains to be explored whether tuning should be adapted to a new meaning ("leniency" or something like that) or eliminated entirely.

Reply:
#8